deid/bene_id_mapping.sql

```diff
@@ -37,10 +37,16 @@ from (
   select /*+ PARALLEL(OUTPATIENT_CONDITION_CODES,12) */ distinct bene_id from OUTPATIENT_CONDITION_CODES union
   select /*+ PARALLEL(BCARRIER_DEMO_CODES,12) */ distinct bene_id from BCARRIER_DEMO_CODES union
   select /*+ PARALLEL(HHA_DEMO_CODES,12) */ distinct bene_id from HHA_DEMO_CODES union
-  select /*+ PARALLEL(HOSPICE_DEMO_CODES,12) */ distinct bene_id from HOSPICE_DEMO_CODES
+  select /*+ PARALLEL(HOSPICE_DEMO_CODES,12) */ distinct bene_id from HOSPICE_DEMO_CODES union
+  select /*+ PARALLEL(MAXDATA_IP,12) */ distinct bene_id from MAXDATA_IP union
+  select /*+ PARALLEL(MAXDATA_LT,12) */ distinct bene_id from MAXDATA_LT union
+  select /*+ PARALLEL(MAXDATA_OT,12) */ distinct bene_id from MAXDATA_OT union
+  select /*+ PARALLEL(MAXDATA_PS,12) */ distinct bene_id from MAXDATA_PS union
+  select /*+ PARALLEL(MAXDATA_RX,12) */ distinct bene_id from MAXDATA_RX union

```

Comment:
Why are MAXDATA_IP ... MAXDATA_RX added here? 3f8c900 says "adde tables for 12caid and 16care " but these are not new; they were in the orginal 2011-2013 data.

Reply:
I see a comment below about "in order for the process to work for the sample data"; is that why?

Reply:
It is true those tables were there for 2011-13, but not for 2014 and 2015, I believe they got removed from master branch when 2014 code got merged.  To keep these changes more organized, we have used tags/release feature. https://github.com/kumc-bmi/grouse/releases